mixer-mixitup.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29728.190
```

Comment:
My VS (2019) keeps doing this, should I manually revert it?

Reply:
Should be fine, I ain't too worried